aikau/src/main/resources/alfresco/services/SiteService.js

```diff
@@ -111,6 +142,20 @@ define(["dojo/_base/declare",
                { label: "create-site.dialog.type.collaboration", value: "site-dashboard" }
             ];
          }
+
+         if (this.additionalSitePresets && this.additionalSitePresets.length)
```

Comment:
Even if I validated that it was an array, I'd still validate that there was something to concatenate. I'll add the additional verification if you insist.